test/parallel/test-coverage-with-inspector-disabled.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const env = Object.assign({}, process.env, { NODE_V8_COVERAGE: '/foo/bar' });
```

Comment:
Style nit:

```suggestion
const env = { ...process.env, NODE_V8_COVERAGE: '/foo/bar' );
```

Reply:
I was trying to maintain consistency with other test cases. 😛 
Fixing this though.